keps/sig-node/1769-memory-manager/kep.yaml

```
@@ -4,11 +4,12 @@ authors:
   - "@bg-chun"
   - "@cezaryzukowski"
   - "@cynepco3hahue"
+  - "@Tal-or" (only for GA graduation)
```

Some validation jobs run tests that parse this file and to ensure that they don't fail we would need something like below:
```suggestion
  - "@Tal-or" #only for GA graduation
```

---

fixed, thanks